### PR TITLE
Release 0.101.3

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Depends on #3052.

The release plan is the following:
1. Merge #3052, update `rel-0.101.3` branch which is based on v0.101.2 + two commits.
2. Merge `rel-0.101.3` into master.
3. Create `v0.101.3` tag pointing to CHANGELOG update for 0.101.3 (00e076b2993cca149ccb61daa35c71bbbdafc11d).
4. Release accoarding to the release notes.

### CODE REVIEW

**Review by cthulhu-rider — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
 
 This document outlines major changes between releases.
 
+## 0.101.3 "Evacuation" (04 Jul 2023)
```
Comment (roman-khimov) [inline CHANGELOG.md:5]: Can't start with E.

File: pkg/core/native/management.go [hunk 2]
```diff
@@ -719,12 +717,10 @@ func (m *Management) emitNotification(ic *interop.Context, name string, hash uti
...
 	l := len(script)
-	offsets := bitfield.New(l)
 	for i := range methods {
 		if methods[i].Offset >= l {
 			return fmt.Errorf("method %s/%d: offset is out of the script range", methods[i].Name, len(methods[i].Parameters))
 		}
-		offsets.Set(methods[i].Offset)
 	}
-	return vm.IsScriptCorrect(script, offsets)
+	return nil
```
Comment (roman-khimov) [inline pkg/core/native/management.go:725]: Well, it's bad. The only reason we have it is delayed 3.6.

File: CHANGELOG.md (hunk 1 above)
Comment (AnnaShaleva) [reply CHANGELOG.md:5]: Ok, it should actually start with U.

Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: pkg/core/native/management.go (hunk 2 above)
Comment (AnnaShaleva) [reply pkg/core/native/management.go:725]: Discussed, we still need our T5 testnet node to be alive until 3.6.0 release and by the time SC coding workshop starts.

**Pull request merged**